openapi/provider.yml

```diff
@@ -0,0 +1,234 @@
+openapi: "3.0.2"
```

Comment:
I copied this file from the spec repo, but we might want to make it a git submodule.